Sources/MeiliSearch/Settings.swift

```diff
@@ -347,21 +347,21 @@ struct Settings {
   func updateDistinctAttribute(
     _ uid: String,
     _ distinctAttribute: String,
-    _ completion: @escaping (Result<Task, Swift.Error>) -> Void) {
+    _ completion: @escaping (Result<TaskInfo, Swift.Error>) -> Void) {
 
     let data: Data
     do {
-      data = try Constants.customJSONEecoder.encode(DistinctAttributePayload(distinctAttribute: distinctAttribute))
+      data = try JSONEncoder().encode(distinctAttribute)
```

Comment:
Why were we able to change `Constants.customJSONEecoder.` here but not in the other methods?

Reply:
I think it's because this distinct-attribute setting just wants a simple string and the other methods require a complex object/array